### ISSUE TITLE
Name Overrides are causing bugs in most Upgrades

### ISSUE THREAD

**Issue by Uncryptical** [Malcolm-Q/LC-LateGameUpgrades#438]
As of using LGU v3.5.7

Name Overrides are causing many bugs for multiple upgrades. I'll try to list as many as possible (based on what i've encountered)

1. Bigger Lungs: Jump Stamina Cost Decrease + Stamina Regen Increase is no longer affected.
2. Back Muscles: Does not work when purchased.
3. Aluminum Coils: Does not work when purchased.
4. Lithium Batteries: It works as intended(?) except when you want to make the battery use indefinite (over 100%)
5. Sick Beats: Unable to purchase upgrade.
6. Sigurd Access: Unable to purchase upgrade.
7. Malware Broadcaster: Unable to purchase upgrade.

These are the upgrades i had problems with. Any other upgrade i did not list are unknown as i personally do not use them in my custom modpack.

TL;DR ALL of these upgrades will work as intended if Name Override is set to False, but will be bugged if you decide to have custom names.
